src/app/components/client/exposure_card/ScanResultCard.tsx

```diff
@@ -197,7 +199,10 @@ export const ScanResultCard = (props: ScanResultCardProps) => {
 
   const attemptCount = scanResult.optout_attempts ?? 0;
   const statusPillNote =
-    scanResult.status === "waiting_for_verification" && attemptCount >= 1
+    props.enabledFeatureFlags?.includes("AdditionalRemovalStatuses") &&
+    !scanResult.manually_resolved &&
```

Comment:
It’s needed, because otherwise we’ll potentially show the “fixed” label together with the removal note. This was unexpected behavior that has been flagged by QA [here](https://mozilla-hub.atlassian.net/browse/MNTOR-3329?focusedCommentId=898213).